skate-plugin/build.gradle.kts

```diff
@@ -92,4 +93,5 @@ dependencies {
   implementation(libs.bugsnag) { exclude(group = "org.slf4j") }
   testImplementation(libs.junit)
   testImplementation(libs.truth)
+  testImplementation("org.mockito:mockito-core:5.5.0")
```

Comment:
We don't use mockito in this project :). Let's stick to fakes and real implementations